### PR TITLE
Consolidated content changes for opposite and same sex (Sweden) 

### PR DESCRIPTION
**Pull request by ikennaokpala**
Trello Story: https://trello.com/c/MaF6tohO/104-marriage-abroad-sweden

This PR consolidates #2462 and #2461.

## Factcheck
* Opposite sex outcomes where user is resident in the UK:
   * [Preview Link](https://smart-answers-pr-2462.herokuapp.com/marriage-abroad/y/sweden/uk/partner_british/opposite_sex)
   * [Link on GOV.UK](https://gov.uk/marriage-abroad/y/sweden/uk/partner_british/opposite_sex)

## Expected changes
* Content changes for opposite outcomes where user is resident in the UK
  * Changes to Legalisation and translation

### Before
screen shot 2016-04-11 at 16 05 13

### After

screen shot 2016-04-11 at 16 04 47

## Factcheck

* Opposite sex marriage outcomes for a resident in Sweden marrying a British/Swedish/National of another country
    * [Preview Link](https://smart-answers-pr-2462.herokuapp.com/marriage-abroad/y/sweden/ceremony_country/partner_british/opposite_sex)
    * [Link on GOV.UK](https://gov.uk/marriage-abroad/y/sweden/ceremony_country/partner_british/opposite_sex)

## Expected changes
* Mostly new content for opposite outcomes where user is resident in Sweden marrying a British/Swedish/National of another country

### Before

screen shot 2016-04-11 at 16 08 50

### After
screen shot 2016-04-11 at 16 08 39

## Factcheck

* Same sex marriage outcomes for a resident in Sweden marrying a British/Swedish/National of another country
    * [Preview Link](https://smart-answers-pr-2462.herokuapp.com/marriage-abroad/y/sweden/ceremony_country/partner_british/opposite_sex)
    * [Link on GOV.UK](https://gov.uk/marriage-abroad/y/sweden/ceremony_country/partner_british/opposite_sex)

## Expected changes
* Mostly new content for same sex outcomes where user is resident in Sweden marrying a British/Swedish/National of another country

### Before

screen shot 2016-04-11 at 16 13 01

### After
screen shot 2016-04-11 at 16 13 33

### CODE REVIEW

**Comment by ikennaokpala** [comment]
This supersedes #2453 and #2461

File: test/integration/smart_answer_flows/marriage_abroad_test.rb [hunk 1]
```diff
@@ -1131,16 +1131,154 @@ def self.translations
...
     setup do
       worldwide_api_has_organisations_for_location('sweden', read_fixture_file('worldwide/sweden_organisations.json'))
       add_response 'sweden'
-      add_response 'ceremony_country'
-      add_response 'partner_other'
-      add_response 'same_sex'
     end
-    should "go to cp or equivalent os outcome" do
-      assert_current_node :outcome_same_sex_civil_partnership
+    context "resident in uk," do
```
Comment (chrisroos) [inline test/integration/smart_answer_flows/marriage_abroad_test.rb]: It's worth noting that this style of nested contexts has been deprecated in favour of "flatter" tests. See https://github.com/alphagov/smart-answers/blob/master/doc/testing.md#deprecated-a-test-using-nested-contexts for more information.

I think the tests are fine in this case given that that's how they're structured in the rest of this file.

**Comment by chrisroos** [comment]
I think the comment for commit titled "Content changes for opposite and same sex (Sweden)" should be updated. It contains some duplicate and outdated information.

I'd probably go with something like:
```
1. Update advice for ceremonies in Sweden when resident in Sweden

Trello card: https://trello.com/c/MaF6tohO

The advice for residents of Sweden wanting to get married in Sweden is now the 
same irrespective or the partner's nationality or sex. This advice is contained 
in the new `outcome_ceremonies_in_sweden_when_residing_in_sweden` outcome. The 
addition to marriage_abroad_services.yml ensures that the same services/fees are
shown for these ceremonies.

The content for this new outcome came from the Google Doc linked from the Trello card.

2. Update advice for ceremonies in Sweden when resident in the UK

Trello card: https://trello.com/c/MaF6tohO

Remove the sentence about exchanging a UK issued CNI from the "Legalisation and 
translation" information.

The content for this change came from the Google Doc linked from the Trello card.

3. Add integration tests for all types of ceremony in Sweden

TODO: Explain why these are necessary.
```

**Comment by chrisroos** [comment]
Aside from updating the commit note, this all looks good to me.

File: test/integration/smart_answer_flows/marriage_abroad_test.rb (hunk 1 above)
Comment (ikennaokpala) [inline test/integration/smart_answer_flows/marriage_abroad_test.rb]: @chrisroos noted 👍

**Comment by ikennaokpala** [comment]
I have updated the comment note and forced pushed it.

```
1. Update advice for ceremonies in Sweden when resident in Sweden

Trello card: https://trello.com/c/MaF6tohO

The advice for residents of Sweden wanting to get married in Sweden is now the 
same irrespective or the partner's nationality or sex. This advice is contained 
in the new `outcome_ceremonies_in_sweden_when_residing_in_sweden` outcome. The 
addition to marriage_abroad_services.yml ensures that the same services/fees are
shown for these ceremonies.

The content for this new outcome came from the Google Doc from the Trello card.

2. Update advice for ceremonies in Sweden when resident in the UK

Trello card: https://trello.com/c/MaF6tohO

Remove the sentence about exchanging a UK issued CNI from the "Legalisation and 
translation" information.

The content for this change came from the Google Doc linked from the Trello card.

3. Add integration tests for all types of ceremony in Sweden

Given the changes highlighted above, the introduction of a new partial 
(outcome_ceremonies_in_sweden_when_residing_in_sweden) and the previous existence
of only one [integration test](https://github.com/alphagov/smart-answers/blob/master/test/integration/smart_answer_flows/marriage_abroad_test.rb#L1134) that covered only one aspect of marriage in Sweden.

It became necessary to cover all types of marriage ceremonies in Sweden:

List of partials involved thus far, for Marriage ceremonies in Sweden:

* outcome_same_sex_civil_partnership
* outcome_ceremonies_in_sweden_when_residing_in_sweden 
* outcome_opposite_sex_marriage_in_consular_cni_countries_when_residing_in_uk_or_ceremony_country
```

**Comment by ikennaokpala** [comment]
I have updated the commit note and forced pushed it.
```
Content changes for opposite and same sex (Sweden)
1. Update advice for ceremonies in Sweden when resident in Sweden

Trello card: https://trello.com/c/MaF6tohO

The advice for residents of Sweden wanting to get married in Sweden is now the
same irrespective or the partner's nationality or sex. This advice is contained
in the new `outcome_ceremonies_in_sweden_when_residing_in_sweden` outcome. The
addition to marriage_abroad_services.yml ensures that the same services/fees are
shown for these ceremonies.

The content for this new outcome came from the Google Doc from the Trello card.

2. Update advice for ceremonies in Sweden when resident in the UK

Trello card: https://trello.com/c/MaF6tohO

Remove the sentence about exchanging a UK issued CNI from the "Legalisation and
translation" information.

The content for this change came from the Google Doc linked from the Trello card.

3. Add integration tests for all types of ceremony in Sweden

Given the changes highlighted above, the introduction of a new partial
(outcome_ceremonies_in_sweden_when_residing_in_sweden) and the previous existence
of only one integration test(https://github.com/alphagov/smart-answers/blob/master/test/integration/smart_answer_flows/marriage_abroad_test.rb#L1134) that covered only one aspect of marriage ceremonies in Sweden.

It became necessary to cover all types of marriage ceremonies in Sweden:

List of partials involved thus far, for Marriage ceremonies in Sweden:

* outcome_same_sex_civil_partnership
* outcome_ceremonies_in_sweden_when_residing_in_sweden
* outcome_opposite_sex_marriage_in_consular_cni_countries_when_residing_in_uk_or_ceremony_country

```

**Pull request merged**